web/crux-ui/docker-compose.yaml

```diff
@@ -1,4 +1,3 @@
-version: '3.3'
```

Comment:
Why did you remove the version? There is no information in the PR about this.

Reply:
Because of deprecation, old versions of compose needed it, but recent version give a warning that it is a deprecated option.

Reply:
Okey, thanks for the clarification. (Missing from the PR description 🧒🏼)